### PR TITLE
Separate out corefoundation build

### PR DESCRIPTION
**Pull request by johnno1962**
Hi @gonzalolarralde, I’ve separated out the foundation build from it’s dependencies if you’re interested.

### CODE REVIEW

**Comment by johnno1962** [comment]
Don’t forget to Squish. It took me an embarising amount of time to find a stupid quoting problem!

File: util/prepare_environment/LICENSE [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+https://github.com/SwiftAndroid/libiconv-libicu-android
+
+https://github.com/apple/swift
+
+https://github.com/apple/swift-corelibs-libdispatch
+
+https://github.com/apple/swift-corelibs-foundation
+
+Built using a fork of the scripts in:
+https://github.com/gonzalolarralde/swifty-robot-environment
```
Comment (ephemer) [inline util/prepare_environment/LICENSE:30]: We should probably take this and related notes out of the PR

**Comment by johnno1962** [comment]
A couple of recent fixes/hacks you may want to use. If we can get the Apple PRs in, a complete scripted build should be possible without having to patch any source.

File: util/prepare_environment/LICENSE (hunk 1 above)
Comment (johnno1962) [inline util/prepare_environment/LICENSE]: Hi @ephemer, I’m confused by this comment. I should take acknowledgement of @gonzalolarralde’s scripts out of any binary release I that I do?

File: util/prepare_environment/060_build_corelibs_libdispatch.sh [hunk 2]
```diff
@@ -27,6 +29,8 @@ pushd $SWIFT_ANDROID_SOURCE
 				--disable-build-tests \
 				--prefix=$SWIFT_INSTALLATION_PATH/usr
 
+		sed -ie "s@-L/tmp/placeholder@-L$SWIFT_INSTALLATION_PATH/usr/lib/swift/android -lswiftCore -L$NDK/toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/arm-linux-androideabi/lib/armv7-a -latomic@" src/Makefile
```
Comment (zayass) [inline util/prepare_environment/060_build_corelibs_libdispatch.sh:32]: I think you can pass it directly to configure without seding. Via LDFLAGS
Like here https://github.com/zayass/swift-android-vagrant/blob/master/corelibs/provision/010_build_corelibs_libdispatch.sh#L16

Comment (johnno1962) [reply util/prepare_environment/060_build_corelibs_libdispatch.sh:32]: Hi Anton, I couldn’t get that to work. It gave me errors about atomic not being avalable.

**Comment by johnno1962** [comment]
Hi @gonzalolarralde, Closing this now, it's probably miles out of date.

**Pull request closed without merging**